bungiesearch/managers.py

```diff
@@ -45,4 +47,7 @@ def __getattr__(self, alias):
         this is only called as a last resort in case the attribute is not found.
         This function will check whether the given model is allowed to use the proposed alias and will raise an attribute error if not.
         '''
+        if alias[0] == '_':
```

Comment:
I recall having that issue when I wanted to support Python 3 ages ago, and never got back to handling it. Anyway, a short comment in the code will help us remember the purpose of it.